lang/gnu-apl/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           select 1.0
+
+name                gnu-apl
+version             1.8
+categories          lang
+platforms           darwin
+license             GPL
+maintainers         {@casouri gmail.com:casouri} openmaintainer
+description         GNU APL is a free interpreter for the programming language APL.
+
+long_description    GNU APL is a free interpreter for the programming language APL. \
+                    The APL interpreter is an (almost) complete implementation of ISO \
+                    standard 13751 aka. Programming Language APL, Extended.
+
+homepage            https://www.gnu.org/software/apl/
+master_sites        gnu:apl
+distfiles           apl-1.8.tar.gz
```

Comment:
Don't set `distfiles` directly. Set the `distname` instead. Also don't hardcode the `version`.

Reply:
Done